shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -0,0 +1,234 @@
+        usedPercentageInfo = String.format("%s%%", usedPercentage);
+      }
+      mStringBuilder.append(String.format("%-20s %-20s %-20s %-20s %-20s%n",
+          workerInfo.getAddress().getHost(),
+          workerInfo.getLastContactSec(),
+          FormatUtils.getSizeFromBytes(capacityBytes),
+          FormatUtils.getSizeFromBytes(usedBytes),
+          usedPercentageInfo));
+
+      sumCapacityBytes += capacityBytes;
+      sumUsedBytes += usedBytes;
+
+      Map<String, Long> totalBytesOnTiers = workerInfo.getCapacityBytesOnTiers();
+      if (totalBytesOnTiers != null) {
+        for (Map.Entry<String, Long> totalBytesTier : totalBytesOnTiers.entrySet()) {
+          String tier = totalBytesTier.getKey();
+          long value = totalBytesTier.getValue();
+          sumTotalBytesOnTiersMap.put(tier, value + sumTotalBytesOnTiersMap.get(tier));
+        }
+      }
+
+      Map<String, Long> usedBytesOnTiers = workerInfo.getUsedBytesOnTiers();
+      if (usedBytesOnTiers != null) {
+        for (Map.Entry<String, Long> usedBytesTier: usedBytesOnTiers.entrySet()) {
+          String tier = usedBytesTier.getKey();
+          long value = usedBytesTier.getValue();
+          sumUsedBytesOnTiersMap.put(tier, value + sumUsedBytesOnTiersMap.get(tier));
+        }
+      }
+    }
+
+    // Print summarized worker capacity information
+    if (options.getWorkerRange().equals(WorkerRange.SPECIFIED)
+        && sumCapacityBytes + sumUsedBytes == 0) {
+      System.out.println(USAGE);
+      throw new InvalidArgumentException("Worker IP addresses are invalid.");
+    }
+
+    mIndentationLevel = 0;
+    print(String.format("Capacity Information for %s Workers: ", options.getWorkerRange()));
+
+    mIndentationLevel++;
+    print("Total Capacity: " + FormatUtils.getSizeFromBytes(sumCapacityBytes));
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> totalBytesTier : sumTotalBytesOnTiersMap.entrySet()) {
+      long value = totalBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + totalBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    print("Used Capacity: "
+        + FormatUtils.getSizeFromBytes(sumUsedBytes));
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> usedBytesTier : sumUsedBytesOnTiersMap.entrySet()) {
+      long value = usedBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + usedBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    if (sumCapacityBytes != 0) {
+      int usedPercentage = (int) (100L * sumUsedBytes / sumCapacityBytes);
+      print(String.format("Used Percentage: " + "%s%%", usedPercentage));
+      print(String.format("Free Percentage: " + "%s%%", 100 - usedPercentage));
+    }
+    print("");
+
+    // Print cached worker information
```

Comment:
remove this comment, it says the same thing as the line of code below it

Reply:
Removed, thanks!